public/style.css

```diff
@@ -3,6 +3,15 @@
     grid-template-columns: 200px auto;
     margin: 0;
     height: 100%;
+    user-select: none;
+}
+
+#editor input {
+    user-select: auto;
+}
+
+#editor #inspector {
+    user-select: text;
 }
```

Comment:
Note: I was considering whether that line should rather be added to the `#inspector` block below. But it's an undecidable style question,  and having it under `#editor` could avoid issues.

Reply:
Ah, I just placed everything at the top to make it easier to change the three rules at once/remember why they were added without having to add comments elsewhere. Feel free to edit anything the way you want.